lib/Twig.php

```diff
@@ -22,12 +22,173 @@ class Twig {
 	public static function init() {
 		$self = new self();
 
-        add_action( 'timber/twig/filters', array( $self, 'add_timber_filters' ) );
-		add_action( 'timber/twig/functions', array( $self, 'add_timber_functions' ) );
-		add_action( 'timber/twig/escapers', array( $self, 'add_timber_escapers' ) );
+        add_action( 'timber/twig/filters', [ $self, 'add_timber_filters' ] );
+		add_action( 'timber/twig/functions', [ $self, 'add_timber_functions' ] );
+		add_action( 'timber/twig/escapers', [ $self, 'add_timber_escapers' ] );
```

Comment:
If we touch this, maybe we can reduce this to only use `timber/twig`, see https://github.com/timber/timber/issues/2284?

```php
add_action( 'timber/twig', [ $self, 'add_timber_filters' ] );
add_action( 'timber/twig', [ $self, 'add_timber_functions' ] );
add_action( 'timber/twig', [ $self, 'add_timber_escapers' ] );
```

Instead, we could use the `timber/twig/default_functions` filters etc. to control what is added?

Reply:
You're absolutely right, both are related and this could be the opportunity to tackle #2284. 

I saw that issue and thought about it too. But I wanted to get your feedback before doing too many changes so I kept this PR in the strict scope of making functions/filters filterable.

Reply:
> I saw that issue and thought about it too. But I wanted to get your feedback before doing too many changes so I kept this PR in the strict scope of making functions/filters filterable.

Let’s follow that approach in https://github.com/timber/timber/pull/2419 and leave this PR as it is 👍 .